### ISSUE TITLE
Getting WARNING: Unsupported BIO ctrl call (76) message

### ISSUE THREAD

**Issue by emanzx** [amberframework/granite#458]
My apps keep throwing this error while granite is connecting to the postgres server to send data. 
```
[Sender] Curent time start sending data 2021-11-10 13:36:43 +08:00
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (73)
...
WARNING: Unsupported BIO ctrl call (73)
WARNING: Unsupported BIO ctrl call (76)
WARNING: Unsupported BIO ctrl call (76)
WARNING: Unsupported BIO ctrl call (76)
[Sender] [Fiber 6] Successfully upload [104] datapoint 2021-11-10 13:40:43 +08:00to timescaledb.
```
I try to look into it but no reference found and the only code I could locate it from crystal lang github which generate the error and it related to ssl implementation 
https://github.com/crystal-lang/crystal/blob/master/src/openssl/bio.cr

anyhelp that can help me

**Comment by Blacksmoke16** [comment]
I would see if you can reproduce it with only https://github.com/will/crystal-pg. This doesn't seem like a Granite issue, but more so something going on in the underlying driver.

**Comment by emanzx** [comment]
I will try.. It may take a while..

**Comment by Blacksmoke16** [comment]
@crimson-knight, you can close this as it'll be fixed in `1.5.0` via https://github.com/crystal-lang/crystal/pull/12034.

**Comment by crimson-knight** [comment]
@Blacksmoke16 thanks for the heads up!
